apps/frontend/core-dapp/src/features/position/Position.tsx

```diff
@@ -149,3 +150,23 @@ export const Position: React.FC<PositionProps> = ({
     </Flex>
   </Flex>
 )
+
+export const PositionSkeleton: React.FC = () => (
+  <Flex justifyContent="start" gap={8} width="100%">
+    <Skeleton circle height={46} width={46} />
```

Comment:
To match the actual logo size

```suggestion
    <Skeleton circle height={48} width={48} />
```